lib/y2_installbase.pm

```diff
@@ -191,6 +192,23 @@ sub save_upload_y2logs {
     $self->investigate_yast2_failure() unless $args{skip_logs_investigation};
 }
 
+sub save_remote_upload_y2logs {
+    my ($self, %args) = @_;
+
+    return if (get_var('NOLOGS'));
+    $args{suffix} //= '';
+
+    type_string 'sed -i \'s/^tar \(.*$\)/tar --warning=no-file-changed -\1 || true/\' /usr/sbin/save_y2logs';
+    send_key 'ret';
+    my $filename = "/tmp/y2logs$args{suffix}.tar" . get_available_compression();
+    type_string "save_y2logs $filename\n";
+    my $uploadname = +(split('/', $filename))[2];
+    my $upname     = ($args{log_name} || $autotest::current_test->{name}) . '-' . $uploadname;
+    type_string " curl --form upload=\@$filename --form upname=$upname " . autoinst_url("/uploadlog/$upname") . "\n";
+    save_screenshot();
+    $self->investigate_yast2_failure();
```

Comment:
I cannot see this line running in your verification simulating failure. Are we sure this will work? I see that there are not `assert_script_run` which could block because ret codes but we might miss something, WDYT?